libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApiTest.kt

```diff
@@ -1262,14 +1262,8 @@ class MapboxRouteLineApiTest {
             StringChecker("[line-progress]"),
             StringChecker("[rgba, 0.0, 0.0, 0.0, 0.0]"),
             DoubleChecker(0.0),
-            StringChecker("[rgba, 136.0, 136.0, 136.0, 1.0]"),
-            DoubleChecker(0.4822003),
-            StringChecker("[rgba, 204.0, 204.0, 204.0, 1.0]"),
-            DoubleChecker(0.4848101),
-            StringChecker("[rgba, 136.0, 136.0, 136.0, 1.0]"),
-            DoubleChecker(0.4904095),
-            StringChecker("[rgba, 204.0, 204.0, 204.0, 1.0]"),
-            DoubleChecker(0.492223),
```

Comment:
Why has this changed? Was the PR fixed as well?

Reply:
Earlier the inactive leg would have been highlighted only on `updateTraveledRouteLine`. Not it's also hightlighted on `updateWithRouteProgress` as a side-effect of highlighting it on `setNavigationRouteLines`. I couldn't find a reason why it's bad, so left it as-is.